VotingApplication/VotingApplication.Web.Api.Tests/Sockets/ChatHubTests.cs

```diff
@@ -173,21 +167,10 @@ public void SendMessage_with_NoMessage_expect_SendError()
         }
 
         [TestMethod]
-        public void SendMessage_with_InvalidUser_expect_SendError()
```

Comment:
But previously you couldn't send a chat message unless you had a user id, which you couldn't get unless you'd entered a user name.

I think the chat window is another area the the UX should probably handle with a just-in-time popup to get the user to enter a name...